docs/docs/tutorial/chapter2/routing-params.md

```diff
@@ -1,13 +1,16 @@
 # Routing Params
 
-Now that we have our homepage listing all the posts, let's build the "detail" page—a canonical URL that displays a single post. First we'll generate the page and route:
```

Comment:
This was already an em-dash, just typed right in there. Since the page contains `<meta charset="UTF-8">` you can just put the actual character inline, no need for the old HTML entity replacement.

Reply:
> page—a

You're right! It was just the monospace font that threw me off, making it look so short